data/prefixes.js

```diff
@@ -659,12 +659,18 @@ f(prefixPointer, browsers =>
 // Text decoration
 let prefixDecoration = require('caniuse-lite/data/features/text-decoration')
 
-f(prefixDecoration, browsers =>
+f(prefixDecoration, { match: /x.*#[235]/ }, browsers =>
+  prefix(['text-decoration-skip', 'text-decoration-skip-ink'], {
+    feature: 'text-decoration',
+    browsers
+  })
+)
```

Comment:
`text-decoration-skip-ink` was kept as-is.

- data from caniuse
- all logic/config same as before